app/views/subscriptions/index.html.haml

```diff
@@ -0,0 +1,15 @@
+- content_for :page_title do
+  = t('.title')
+
+- content_for :content do
+  #container.container-fluid
+    %h1= t('.title')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.